Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -89,17 +98,17 @@ public void Roslyn_Settings_ValidSetup_MergeWithExistingSettings_AbsolutePath()
             string targetDir = result.ProjectStateAfterBuild.GetPropertyValue(TargetProperties.TargetDir);
             string expectedErrorLog = Path.Combine(targetDir, ErrorLogFileName);
 
-            AssertExpectedErrorLog(result, expectedErrorLog);
-            AssertExpectedAdditionalFiles(result, GetDummySonarLintXmlFilePath());
+            AssertExpectedAnalysisProperties(result, expectedErrorLog, sourceRulesetFilePath, GetDummySonarLintXmlFilePath());
             BuildAssertions.AssertWarningsAreNotTreatedAsErrors(result);
 
-            string mergedRulesetFilePath = AssertMergedResultFileExists(result, sourceRulesetFilePath);
-            RuleSetAssertions.AssertExpectedIncludeFilesAndDefaultAction(mergedRulesetFilePath, "c:\\existing.ruleset");
+            // Check the analyzer properties are set as expected
+            List<string> expectedAnalyzers = new List<string>(GetSonarLintAnalyzerFilePaths());
+            AssertExpectedItemValuesExists(result, TargetProperties.AnalyzerItemType, expectedAnalyzers.ToArray());
         }
 
         [TestMethod]
-        [Description("Checks the ruleset and analyzers list are correctly merged with existing settings")]
-        public void Roslyn_Settings_ValidSetup_MergeWithExistingSettings_RelativePath()
+        [Description("Checks the ruleset and analyzers list correctly override existing settings")]
+        public void Roslyn_Settings_ValidSetup_OverrideExistingSettings_RelativePath()
```

Comment:
We probably don't really need separate tests for absolute and relative existing ruleset paths as we overwrite them, but I've left both tests anyway.